cmd/minishift/cmd/delete.go

```diff
@@ -40,8 +40,6 @@ func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
-	setSubcriptionManagerParameters()
-
```

Comment:
So are you saying we don't have the username/password flags on stop/delete at all anymore? So they won't get passed to un-register either?